NetKAN/QuodiosKermansMineToEarn.netkan

```diff
@@ -0,0 +1,7 @@
+{
+    "$kref": "#/ckan/spacedock/951",
+    "identifier": "QuodiosKermansMineToEarn",
+    "spec_version": "v1.4",
+    "license": "GPLv3",
```

Comment:
Just a quick note - There's no one from Spacedock watching these submissions. We CKAN folk have to fix them up.